packages/checkbox/src/CheckboxEditor.tsx

```diff
@@ -86,29 +104,44 @@ export function CheckboxEditor(props: CheckboxEditorProps) {
           setValue(newValues);
         };
 
+        const invalidValues = getInvalidValues(field, values, options);
+        const mergedOptions = [...options, ...invalidValues];
+
```

Comment:
Thoughts on something like below?

```
const { options, invalid } = partitionOptions(field, values, options)

return (
  <Form ...>
    {options.map(...)}
    {invalidOptions.map(...)}
  </Form>
)
```

Reply:
You might even be able to do the Checkbox render inside of the partitionOptions command (or whatever name you give it)

Reply:
This could work but I don't understand why this approach. Could you please shed some light?

Reply:
I think this approach is repetitive. What benefit does it provide?
````
options.map((item) => (<CheckboxField/>))
invalidOptions.map((item)) => (<CheckboxField/>)
```

Reply:
The main thing is that we don't need to munge the two lists together when iterating, instead we can just render the valid options and then render the invalidOptions right below

Reply:
Is there any disadvantage we are trying to avoid by not copying both lists into one array and rending them as such?